msteams-platform/concepts/deploy-and-publish/appsource/publish.md

```diff
@@ -90,7 +90,7 @@ If your app is production ready, you can begin the process of getting it listed
 
   1. [Create your Partner Center account](prepare/create-partner-center-dev-account.md) well in advance. If you run into any challenges with your [Partner Center account](prepare/create-partner-center-dev-account.md), create a [support ticket](/azure/marketplace/partner-center-portal/support).
 
-  1. Review the [store validation guidelines](prepare/teams-store-validation-guidelines.md) again to ensure that your app is in alignment with the store requirements. This helps reduce the number of issues observed in your app and so, the time taken to approve your app.
+  1. Review the [store validation guidelines](prepare/teams-store-validation-guidelines.md) again to ensure that your app is in alignment with the store requirements, which helps reduce the number of issues observed in your app and so, the time taken to approve your app.
```

Comment:
Please undo this combining of sentences into such long ones. Edit as required but keep the sentences short. 

CC @v-habibaumme , @v-psambrani